packages/unmock-core/src/service/state/interfaces.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  HTTPMethod,
+  IStateInput,
+  Operation,
+  Paths,
+  Schema,
+} from "../interfaces";
+
+export interface IStateUpdate {
+  stateInput: IStateInput;
+  serviceName: string;
+  paths: Paths;
+  /**
+   * Complements the endpoint given in IStateInput, by redirecting to the correct endpoint in Schema.
+   * This happens when an endpoint consists of variables, for example.
```

Comment:
I think an example would generally help to describe what the schema endpoint is. What's the difference between endpoint and a path in our code?